ReflectometryServer/ioc_driver.py

```diff
@@ -7,6 +7,9 @@
 
 logger = logging.getLogger(__name__)
 
+# Tolerance within which the position will be declared out of the beam. 1 mm tolerance.
+TOLERANCE_ON_OUT_OF_BEAM_POSITION = 1
```

Comment:
Is this really a constant everywhere? Should this not be part of the configuration?